metaflow/cli.py

```diff
@@ -649,16 +649,27 @@ def run(obj,
               default=DEFAULT_RUN_NAME,
               help="name assigned to the new KFP run"
               )
+@click.option('--namespace',
+              'namespace',
+              default=KFP_SDK_NAMESPACE,
+              help="namespace of your run in KFP."
+              )
```

Comment:
You'll likely also need an `api-namespace` option too as we run the API service in one namespace but actually run the pipelines in another. So you'll need the `api_namespace` to be fed into the `kfp.Client` constructor so the `kfp` SDK knows where to actually communicate to but you'll feed in `namespace` literally everywhere else as that's the pipeline/team/project specific namespace :) 

`api-namespace` should never really be used and it's best to just default it to `kubeflow` as that's the default namespace the Pipelines API service lives in :)

Reply:
Got it, thanks for the heads up. Resolved.